python/src/py_opcua_variant.h

```diff
@@ -42,6 +42,26 @@ list ToList(const std::vector<SourceType> objects)
   return result;
 }
 
+// This specialisation is needed because on OS-X 10.12 clang-802.0.42
+// with boost 1.59.0 gives the following compile error:
+// /opt/local/include/boost/python/converter/arg_to_python.hpp:209:9:
+//   error: no matching constructor for initialization of
+//      'boost::python::converter::detail::arg_to_python_base'
+//      : arg_to_python_base(&x, registered<T>::converters)
+template <>
+list ToList(const std::vector<bool> objects)
+{
+  list result;
+
+  for (auto obj : objects)
+    {
+      bool tmp = obj;
```

Comment:
Do you mind removing this one and create a separate PR? I'm setting things up to look into it on a mac. Pending CI, patches look good - thank you!